ui/raidboss/popup-text.js

```diff
@@ -396,6 +404,7 @@ class PopupText {
       // * user disabled (play nothing)
       // * if tts options are enabled globally or for this trigger:
       //   * user trigger tts override
+      //   * groupTTS entries in the trigger
```

Comment:
I think this ordering is not correct based on what you have below.  I think groupTTS > user trigger tts override (triggerOptions.TTS).

Can you also list user trigger group tts override here in this list, since that's also a thing you added?

Reply:
Thanks for the clarification.  I think this all seems reasonable to me.